Fraction.py

```diff
@@ -31,13 +32,23 @@ def gcd(a, b):
             return Fraction.gcd(b, a % b)
 
     def get_numerator(self):
-        # TODO
-        pass
+
+        gcd = self.gcd(self.numerator,self.denominator)
+        lowest_numerator = self.numerator // gcd
+        return str(lowest_numerator)
+
 
     def get_denominator(self):
-        # TODO
-        pass
+        gcd = self.gcd(self.numerator,self.denominator)
+        lowest_denominator = self.denominator // gcd
+        return str(lowest_denominator)
 
     def get_fraction(self):
-        # TODO
-        pass
+        numerator = int(self.get_numerator())
+        denominator = int(self.get_denominator())
+        if denominator == 1:
+            return str(numerator)
+        if denominator < 0:
```

Comment:
no more need for this check, numerator should be the only one that can be negative.